packages/asc-ui/src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import Input from './Input'
+
+const inputRef = React.createRef()
+
+const InputComponent: React.FC<{}> = () => (
+  <>
+    <Input
+      onChange={action('onChange')}
+      onKeyDown={action('onKeyDown')}
+      inputRef={inputRef}
```

Comment:
Why do we need to pass the ref? This case seems a bit like a bad practice... https://reactjs.org/docs/refs-and-the-dom.html

Reply:
good point, I am tryin to refactor that now